README.md

```diff
@@ -377,6 +377,13 @@ const multiTiff = await fromUrls(
 
 Geotiff.js supports the use of [`AbortController`s](https://developer.mozilla.org/en-US/docs/Web/API/AbortController). Calls to `getRasters`, `readRGB` and `getTileOrStrip` will throw an `Error` with name `AbortSignal` similar to the browser's `fetch` behavior.
 
+In order to enable this, when you make the tiff source, you need to set the `cacheSize` parameter to `0` due to cache consistency issues - otherwise, once the cache becomes full,
+it will lose its consistency.
+
+```javascript
+const tiff = await fromUrl(source, { cacheSize: 0 });
```

Comment:
Another option here would be to remove the usage of the `LRUCache` and switch to a `Map` if this is unclear.